playbooks/roles/streisand-gateway/tasks/main.yml

```diff
@@ -101,43 +101,75 @@
   command: base64 --wrap=0 {{ openssl_ca_certificate }}
   register: streisand_certificate_data_uri
 
-  # TODO:
-  #   Add to CI testing https://github.com/jlund/streisand/issues/643
```

Comment:
Can you re-add this TODO comment? I think it still applies since this block is gated on the CI var.